src/builder.rs

```diff
@@ -166,6 +164,16 @@ impl Builder {
         self
     }
 
+    pub fn enable_tokio_io(&mut self) -> &mut Self {
+        self.config.enable_tokio_io = true;
+        self
+    }
+
+    pub fn disable_tokio_io(&mut self) -> &mut Self {
```

Comment:
nit: I don't think we'll need to switch a builder back and forth so one method seems fine. If you have a need for it consider using a single method that takes a bool.